tests/rptest/services/admin_ops_fuzzer.py

```diff
@@ -539,4 +539,20 @@ def stop(self):
         assert self.error is None, f"Encountered an error in admin operations fuzzer: {self.error}"
 
     def wait(self, count, timeout):
-        wait_until(lambda: self.executed > count, timeout, 2)
+        def check():
+            # Drop out immediately if the main loop errored out.
+            if self.error:
+                self.redpanda.logger.error(
+                    f"wait: terminating for error {self.error}")
+                raise self.error
+
+            if self.executed > count:
```

Comment:
It is the same as in the original, but I think this should be >=?

Reply:
hmm, I guess it doesn't really matter as the test is waiting for an arbitrary number anyway, but yeah it's more intuitive to be >=, I'll update this.